src/test/java/com/solvd/carina/demo/CarinaDocsTest.java

```diff
@@ -0,0 +1,165 @@
+package com.solvd.carina.demo;
+
+import com.solvd.carina.demo.gui.pages.desktop.CarinaDocsHomePage;
+import com.zebrunner.agent.core.annotation.TestLabel;
+import com.zebrunner.carina.core.IAbstractTest;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import com.zebrunner.carina.core.registrar.tag.Priority;
+import com.zebrunner.carina.core.registrar.tag.TestPriority;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Point;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class CarinaDocsTest implements IAbstractTest {
+
+    @Test
+    @MethodOwner(owner = "qpsdemo")
+    @TestPriority(Priority.P3)
+    @TestLabel(name = "feature", value = { "web", "regression" })
+    public void testZebrunnerLogo() {
+        CarinaDocsHomePage carinaDocsHomePage = new CarinaDocsHomePage(getDriver());
+        carinaDocsHomePage.open();
+        Assert.assertTrue(carinaDocsHomePage.isPageOpened(), "Home page is not opened");
+
+        ExtendedWebElement zebrunnerImage = carinaDocsHomePage.getZebrunnerImage();
+        Point point = zebrunnerImage.getLocation();
+
+        String expectedImgUrl = "http://zebrunner.github.io/carina/assets/logo.svg";
+        String actualImgUrl = zebrunnerImage.getAttribute("src");
+
+        Assert.assertEquals(actualImgUrl, expectedImgUrl, "The image url should be: http://zebrunner.github.io/carina/assets/logo.svg");
+        Assert.assertTrue(point.getX() < 300, "The zebrunner logo should be on the left side of the header");
+
+        zebrunnerImage.click();
+        String expextedNewUrl = "http://zebrunner.github.io/carina/";
+        String actualNewUrl = getDriver().getCurrentUrl();
+        Assert.assertEquals(actualNewUrl, expextedNewUrl, "Clicking on the logo should return back to the home documentation page");
```

Comment:
```suggestion
        Assert.assertEquals(
            getDriver().getCurrentUrl(), 
            "http://zebrunner.github.io/carina/",
            "Clicking on the logo should return back to the home documentation page");
```
It is not necessary to split it here.

Reply:
Shortened my tests to not split